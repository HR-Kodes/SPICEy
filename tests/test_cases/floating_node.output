Error: Node 2 is floating (no DC path to ground).
