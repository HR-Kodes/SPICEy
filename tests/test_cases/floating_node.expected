{
  "error_code": 6
}